<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="sabari" failures="0" tests="6" name="com.financeDomain.adfdatafinance.Scripts.TestCase2" time="0.488" errors="1" timestamp="2022-10-30T09:30:58 IST" skipped="5">
  <testcase classname="com.financeDomain.adfdatafinance.Scripts.TestCase2" name="testcase2" time="0.000">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <system-out/>
  <testcase classname="com.financeDomain.adfdatafinance.Scripts.TestCase2" name="testcase2" time="0.000">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <system-out/>
  <testcase classname="com.financeDomain.adfdatafinance.Scripts.TestCase2" name="configBS" time="0.488">
    <error message="Unknown database &#039;testing&#039;" type="java.sql.SQLSyntaxErrorException">
      <![CDATA[java.sql.SQLSyntaxErrorException: Unknown database 'testing'
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:120)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:829)
at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:449)
at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:242)
at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:198)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:683)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:230)
at com.financeDomain.adfdatafinance.GenericLibraries.FileUtilities.getDataDb(FileUtilities.java:79)
at com.financeDomain.adfdatafinance.GenericLibraries.BaseClass.configBS(BaseClass.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- configBS -->
  <system-out/>
  <testcase classname="com.financeDomain.adfdatafinance.Scripts.TestCase2" name="testcase2" time="0.000">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <system-out/>
  <testcase classname="com.financeDomain.adfdatafinance.Scripts.TestCase2" name="testcase2" time="0.000">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <system-out/>
  <testcase classname="com.financeDomain.adfdatafinance.Scripts.TestCase2" name="testcase2" time="0.000">
    <skipped/>
  </testcase> <!-- testcase2 -->
  <system-out/>
</testsuite> <!-- com.financeDomain.adfdatafinance.Scripts.TestCase2 -->
